mne/viz/raw.py

```diff
@@ -810,3 +810,41 @@ def plot_raw_psd_topo(raw, tmin=0., tmax=None, fmin=0, fmax=100, proj=False,
     except TypeError:  # not all versions have this
         plt_show(show)
     return fig
+
+
+def plot_sensors(info):
+    """Plot sensor positions in 3D.
+
+    Parameters
+    ----------
+    info : Instance of Info
+        Info structure containing the channel locations.
+
+    Returns
+    -------
+    fig : instance of matplotlib figure
+        Figure containing the 3D sensor topography.
+    """
+    import matplotlib.pyplot as plt
+    from mpl_toolkits.mplot3d import Axes3D
+    pos = np.asarray([ch['loc'][:3] for ch in info['chs']])
```

Comment:
...and actually, each channel type should probably be shown with specific colors. It would be good to unify this with `plot_trans` while we're at it.